handbook/engineering/distribution/goals.md

```diff
@@ -93,6 +93,31 @@ We will initially focus on reducing the time it takes to collect troubleshooting
   - ~~Create code to bootstrap new projects.~~ Done: 3.21
   - Document project and folder usage guidelines.
 
+### Site admins use reliable methods like Docker Compose or Kubernetes for production deployments
+
+Many customers of Sourcegraph today are still running a single-container `sourcegraph/server` deployment in production. In 2019 we began advising all new deployments that this deployment option is _not_ for production use because it has no proper resource isolation and as such when it falls over it is impossible to debug, leading to painstakingly urgent migrations to better deployment types and frustrated/angry customers. We would like to get to a world where all production instances of Sourcegraph use reliable deployment methods like Docker Compose or Kubernetes.
+
+- **Owner**: TBD
+- **Status**: [RFC 263 REVIEW: Single-container deployments are for demos only](https://docs.google.com/document/d/1GPypas4ZUZIw346EcNDM1up2OOQFyPpEzA3-0glPEMY/edit)
+- **Outcomes**:
+  - Customer deployments become more reliable in performance, stability, uptime, etc.
+  - No customers are using the unreliable `sourcegraph/server` for their production deployment.
+  - Customers upgrade to Docker Compose or Kubernetes deployments without any major trouble.
+- **Milestones**:
+  - ~~RFC creation.~~
+  - RFC approval.
+  - ...TBD...
+
+### Upgrades across multiple Sourcegraph versions should be easier
```

Comment:
I copied this over from https://github.com/sourcegraph/about/pull/1104/files - I think this is a valuable goal worth calling out still here (even if it turns out to be something I work on)

Reply:
```suggestion
### Support upgrading across multiple Sourcegraph versions
```
I think the current header could be mixed with `Upgrades between releases are easy to perform`.